### PR TITLE
Update state, secrets examples. Fix state toString. Ignore sdk-autogen in code coverage.

### PR DESCRIPTION
**Pull request by mukundansundar**
# Description

Update state example. 

Fix state toString.

Ignore sdk-autogen in code coverage.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #341 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

**Review by artursouza — DISMISSED** [review]
LGTM just two

File: examples/src/main/java/io/dapr/examples/secrets/README.md [hunk 1]
```diff
@@ -61,18 +61,23 @@ Dapr's API for secret store only support read operations. For this sample to run
 vault login myroot
 ```
 
-> Note: If you get `http: server gave HTTP response to HTTPS client` make sure the local vault address is set `export VAULT_ADDR=http://127.0.0.1:8200/`
+> Note: If you get `http: server gave HTTP response to HTTPS client` make sure to se the local vault address as  `export VAULT_ADDR=http://127.0.0.1:8200/`
```
Comment (tcnghia) [inline examples/src/main/java/io/dapr/examples/secrets/README.md:64]:
```suggestion
> Note: If you get `http: server gave HTTP response to HTTPS client` make sure to set the local vault address as  `export VAULT_ADDR=http://127.0.0.1:8200/`
```

**Review by tcnghia — CHANGES_REQUESTED** [review]

**Review by artursouza — APPROVED** [review]

File: examples/src/main/java/io/dapr/examples/secrets/README.md [hunk 2]
```diff
@@ -61,18 +61,23 @@ Dapr's API for secret store only support read operations. For this sample to run
...
 
-> Note: If you get `http: server gave HTTP response to HTTPS client` make sure the local vault address is set `export VAULT_ADDR=http://127.0.0.1:8200/`
+> Note: If you get `http: server gave HTTP response to HTTPS client` make sure to set the local vault address as  `export VAULT_ADDR=http://127.0.0.1:8200/`
 
 2. Create secret (replace `[my favorite movie]` with a title of our choice):
 ```bash
 vault kv put secret/dapr/movie title="[my favorite movie]"
 ```
 
+3. Create random secret:
```
Comment (tcnghia) [inline examples/src/main/java/io/dapr/examples/secrets/README.md:71]:
```suggestion
3. Create a second secret, which we will not include in the `allowedSecrets` list:
```

File: examples/src/main/java/io/dapr/examples/secrets/README.md [hunk 3]
```diff
@@ -61,18 +61,23 @@ Dapr's API for secret store only support read operations. For this sample to run
...
+```bash
+vault kv put secret/dapr/randomKey testVal="value"
```
Comment (tcnghia) [inline examples/src/main/java/io/dapr/examples/secrets/README.md:73]: May be use a name like `disallowedKey`? and `testVal="to demonstrate exclusion from secret allow list"` to be more descriptive?

**Review by tcnghia — CHANGES_REQUESTED** [review]
The name `randomKey` was unclear to me when I went through the doc. So may be a rename to be more descriptive?

Thanks

File: examples/src/main/java/io/dapr/examples/secrets/README.md (hunk 3 above)
Comment (mukundansundar) [reply examples/src/main/java/io/dapr/examples/secrets/README.md:73]: A specific key is not disallowed here ... rather only a specific key is allowed.

**Review by tcnghia — APPROVED** [review]

**Pull request merged**